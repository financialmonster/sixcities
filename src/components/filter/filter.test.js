import React from 'react';
import renderer from 'react-test-renderer';

import Filter from './filter.jsx';

it(`Correctly renders the Filter component `, () => {
    const tree = renderer.create(<Filter />)
        .toJSON();

    expect(tree).toMatchSnapshot();
});
