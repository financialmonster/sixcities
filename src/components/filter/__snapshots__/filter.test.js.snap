// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Correctly renders the Filter component 1`] = `
Array [
  <h2
    className="visually-hidden"
  >
    Places
  </h2>,
  <b
    className="places__found"
  >
    3
     places to stay in 
    Paris
  </b>,
  <form
    action="#"
    className="places__sorting"
    method="get"
  >
    <span
      className="places__sorting-caption"
    >
      Sort by 
    </span>
    <span
      className="places__sorting-type"
      onClick={[Function]}
      tabIndex="0"
      title="Open the menu"
    >
      Popular
      <svg
        className="places__sorting-arrow"
        height="4"
        width="7"
      >
        <use
          xlinkHref="#icon-arrow-select"
        />
      </svg>
    </span>
  </form>,
]
`;
